library/src/morpheus-master/sass/tool.scss

```diff
@@ -96,3 +96,6 @@
 @import "modules/theme-switcher/base";
 @import "modules/multiselect/base";
 @import "modules/sakai-options-menu/sakai-options-menu";
+
+// TODO Master merge: Remove this import
+@import "modules/b5-polyfill";
```

Comment:
Other tools can benefit from this, right? Would be good to shout at the S2U channel, I like this!

Reply:
Yes : )

Reply:
Put a bigger comment explaining this for other developers!

Reply:
Added, at the polyfill file. Let's link to the file on the s2u-22x branch after merging